packages/components/src/components/status-badge/status-badge.scss

```diff
@@ -44,7 +43,6 @@
             border-color: var(--status-danger);
             color: $color-status-danger;
             font-weight: bold;
-            text-decoration: underline;
```

Comment:
we need this in tradershub dashboard @Dhruv-deriv .
i have added a class to override it in verification modal.. should be working in feature branch. can you help to recheck?

Reply:
on it 👍